dropwizard-example/README.md

```diff
@@ -9,7 +9,7 @@ Included with this application is an example of the optional DB API module. The
 the features available in [Hibernate](http://hibernate.org/), along with demonstrating how these are used from within
 Dropwizard.
 
-This database example is comprised of the following classes:
+This database example comprises of the following classes:
```

Comment:
I'm fully aware that "comprises of" is a mine field (See [npr clip](https://www.npr.org/2015/03/12/392568604/dont-you-dare-use-comprised-of-on-wikipedia-one-editor-will-take-it-out)), so please correct me if I'm wrong, but is this more correct?

```suggestion
This database example comprises the following classes:
```

Reply:
Yes, I think that reads better. Amended.